src/wallet/rpcwallet.h

```diff
@@ -7,10 +7,18 @@
 
 #include <span.h>
 
+#include <memory>
+#include <string>
+#include <vector>
+
 class CRPCCommand;
+class CWallet;
+struct WalletContext;
 
 Span<const CRPCCommand> GetWalletRPCCommands();
 
+std::tuple<std::shared_ptr<CWallet>, std::vector<bilingual_str>> LoadWalletHelper(WalletContext& context, UniValue load_on_start_param, const std::string wallet_name);
```

Comment:
What would happen if this was moved to wallet/rpc/util?

Reply:
Yeah I was thinking about it but also conscious of @ryanofsky's idea to potentially split further into load.cpp and thought it would be better there. I can move it in a follow-up if preferred.